solidity/test/isms/OPStackIsm.t.sol

```diff
@@ -128,13 +138,15 @@ contract OPStackIsmTest is Test {
         vm.selectFork(mainnetFork);
 
         bytes memory encodedHookData = abi.encodeCall(
-            OPStackIsm.verifyMessageId,
-            (address(this).addressToBytes32(), messageId)
+            AbstractMessageIdAuthorizedIsm.verifyMessageId,
+            (messageId)
         );
 
         uint40 nonce = ICanonicalTransactionChain(L1_CANNONICAL_CHAIN)
             .getQueueLength();
 
+        l1Mailbox.updateLatestDispatchedId(messageId);
+
         vm.expectEmit(true, true, true, false, L1_MESSENGER_ADDRESS);
         emit SentMessage(
```

Comment:
imo this isn't an idiomatic unit test because we are testing for behavior outside of our codebase

Reply:
I don't understand.
What's the alternative here? the `testMailbox` is serving as a mock.

Reply:
iiuc the `SentMessage` is from optimism contract implementation which could change in future out from under us

Reply:
you could use `expectCall` or whatever to test our usage of the messenger interface